templates/internship_modification_student.html

```diff
@@ -41,11 +41,11 @@ <h2>{% trans 'internships' %} : {{ student.person.last_name }} {{ student.person
         <div class="panel-body">
             <div class="row">
                 <ul class="nav nav-tabs">
-                    {% for internship in internships %}
-                      {% if internship.speciality != None %}
-                        <li role="presentation" class="active"><a href="{% url 'specific_internship_student_modification' cohort_id=cohort.id student_id=student.id internship_id=internship.id speciality_id=internship.speciality_id %}">{{ internship.name }}</a></li>
+                    {% for internship_item in internships %}
+                      {% if internship_item.speciality != None %}
+                        <li role="presentation" class="{% if internship_item.id == internship.id %} active {% endif %}"><a href="{% url 'specific_internship_student_modification' cohort_id=cohort.id student_id=student.id internship_id=internship_item.id speciality_id=internship.speciality_id %}">{{ internship_item.name }}</a></li>
```

Comment:
Small fix to do: `speciality_id=internship_item.speciality_id` instead of `speciality_id=internship.speciality_id`. Thanks in advance.